School.js

```diff
@@ -12,37 +14,46 @@ class School {
   /**
    * Add class to classes
    * 
-   * @param {string} name - Name of the class
-   * @param {string} teacher - Name of instructor 
-   * @return {Class} Class object
+  //  * @param {string} name - Name of the class
+  //  * @param {string} teacher - Name of instructor 
+  //  * @return {Class} Class object
    */
   addClass(name, teacher) {
     let newClass = new Class(name, teacher);
-    this.classes[name] = newClass;
+    return this.classes[name] = newClass;
```

Comment:
just `return newClass` after this line. Assign the key-value pair and return newClass